Makefile

```diff
@@ -86,6 +86,11 @@ ifdef NYDUS_TEST_VIRTIOFS
 	RUST_BACKTRACE=1 cargo test $(VIRIOFS_COMMON) --bin nydusd -- --nocapture --test-threads=8
 endif
 
+macos-ut:
```

Comment:
Can you enable the smoke test on macos as well? Just so that we know the code actually works roughly.

Reply:
As the fuse-backend-rs, github action with macos can not use macfuse, so the smoke test can not run.

Reply:
Ok, fair enough. Let's just keep it w/o e2e tests and hopefully the UT can cover most cross-platform differences.